docs/modules/vcftools.md

```diff
@@ -10,6 +10,15 @@ This module parses the outputs from VCFTools' various commands:
 * `relatedness2`
   * Plots a heatmap of pairwise sample relatedness.
   * Not to be confused with the similarly-named command `relatedness`
+* `TsTv-by-count`
+  * Plots the transition to transversion ratio as a function of 
+    alternative allele count (using only bi-allelic SNPs).
+* `TsTv-by-qual`
+  * Plots the transition to transversion ratio as a function of SNP 
+    quality threshold (using only bi-allelic SNPs).
+* `TsTv-summary`
+  * Plots a bargraph of the summary counts of each type of transition and 
+    transversion SNPs. 
 
```

Comment:
Please add a subsection here explaining that input files can be very large and that users should increase `config.log_filesize_limit` if they're missing input files (with a disclaimer that this can make MultiQC very slow). You can link to this part of the docs for more information: http://multiqc.info/docs/#big-log-files